### ISSUE TITLE
Focus follow mouse delay

### ISSUE THREAD

**Issue by Andy3153** [hyprwm/Hyprland#3196]
### Description

What I miss from DE's that implement 'Focus follows mouse' is the ability to set a delay for how much the mouse cursor has to hover over the window until it receives focus. This would be very useful to me as I find myself quite a lot of times using window and statusbar gaps in order to escape giving the focus to an unintended window, for example to then use my 'close' and 'maximize' button custom modules I made for myself in Waybar, to try to mimic my previous KDE layout.

**Comment by Aman9das** [comment]
Yeah mouse hover delay when changing focus would benifit my layout too, whis is sort of a train of windows, where the windows are always centered.

Moving the mouse changes focus to the next one, which moves to the center and now mouse is on the next window, and it moves to the center until all the windows are switched through. 

TLDR: In case focusing moves the windows around dynamically, instantaneously changing the focus on hover causes chaos as the windows move into the mouses way and move around.
